build/build/paths.yaml

```diff
@@ -100,6 +101,9 @@
         docker-entrypoint.sh:
         Dockerfile:
     simple-library-server/:
+  std-bits/:
+    benchmarks/:
+      bench-report.xml:
```

Comment:
This is where `sbt std-benchmarks/bench` saves its output. In the same format as `sbt runtime/bench`. After the benchmarks are run, I would like to upload this file as an artifact to the GH Run.

Reply:
Which docs do you mean? This is a CI internal stuff that just instructs the CI job where to find the files that should be uploaded as artifacts.